zstacklib/zstacklib/utils/sanlock.py

```
@@ -515,3 +680,48 @@ def get_watchdog_fire_timeout():
+        if match:
+            vg_name = match.group(1)
+            lockspace_name = "lvm_" + vg_name
+            resource_name = "VGLK"
+            break
+        match = re.search(r'LV (\S+)/(\S+) lock failed: sanlock lease needs repair', line)
+        if match:
+            vg_name = match.group(1)
+            lockspace_name = "lvm_" + vg_name
+            resource_name = match.group(2)
+            break
+        match = re.search(r'(Global lock failed|Skipping global lock): sanlock lease needs repair', line)
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)